_includes/code/components/language-selector--three-or-more.html

```diff
@@ -1,3 +1,3 @@
-{% capture language-selector--three-or-more %}
-  {% library_component usa-language-selector--three-or-more %}
-{% endcapture %}{{ language-selector--three-or-more | strip }}
\ No newline at end of file
+{% capture language-selector--multiple %}
```

Comment:
Updated references here to match the naming from USWDS. It would be beneficial if the names of the variants were consistently applied.